io.github.nokse22.inspector.json

```diff
@@ -0,0 +1,40 @@
+{
+    "app-id" : "io.github.nokse22.inspector",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "inspector",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=session-bus"
```

Comment:
this is not allowed.

Reply:
I found out after adding it, i am trying to find out what permission i need be able to run commands with subprocess.Popen the app runs commands like `lsusb` and displays the results neatly.

`talk-name=org.freedesktop.Flatpak` worked in the app, but gave an error when built here.
   ```
 "errors": [
        "finish-args-flatpak-spawn-access"
```

Reply:
You need to [add a exception here](https://github.com/flathub/flatpak-builder-lint/blob/master/flatpak_builder_lint/staticfiles/exceptions.json).

btw:
We should block `--socket=session-bus` with flatpak-builder-lint by default.

Reply:
Thank you very much!

Reply:
it is already blocked.